proto/iotics/api/common.proto

```diff
@@ -165,16 +165,13 @@ message SubscriptionHeaders {
 
 }
 
-// HostID is a unique host identifier.
-message HostID {
-  // Host Identifier string representation
-  string value = 1;
-}
-
-// TwinID is a unique twin identifier.
+// TwinID is the virtual representation of a (physical, purely virtual or hybrid) device,
+// is only ever located in the host it was created.
 message TwinID {
```

Comment:
Just throwing this into the wild: `id` in here might be confusing. Have you considered just `twin`?